Stable-Branch-Strategy.md

```diff
@@ -0,0 +1,96 @@
+Branch and release maintenance for Kata Containers project
+
+# Overview
+
+As detailed in the [release documentation](https://github.com/kata-containers/documentation/blob/master/Releases.md),
+the Kata Containers project makes use of semantic versioning.  A release version is described
+by MAJOR.MINOR.PATCH.  Early in the project, we will introduce many new features which will require
+minor version changes as well as bug fixes.  To facilitate a stable user environment, Kata will
+begin to provide stable branch based releases as well as a master branch release.
+
+## Stable branch patch criteria
+
+No new features should be introduced to stable branches.  The intention is to limit risk to users,
+providing only bug and security fixes.
+
+## Branch Management
+Kata Containers will maintain two stable release branches in addition to the master branch.
+Once a new MAJOR or MINOR release is created from master, a new stable branch will be created for
+the prior MAJOR or MINOR release and the older stable branch will no longer be maintained.
+
+A couple of examples are provided to help clarify this process.
+
+### New bug fix introduced
+
+A bug fix is submitted against the runtime which does not introduce new inter-component dependencies.
+This fix will be applied to both the master and stable branch.
+
+| Branch | Original version | New version |
+|--|--|--|
+| `master` | `1.3.rc0` | `1.3.rc1` |
+| `stable-1.2` | `1.2.0` | `1.2.1` |
+| `stable-1.1` | `1.1.2` | `1.1.3` |
+
+
+### New release made feature or change which  adds new inter-component dependency
+
+A new feature is introduced which adds a new inter-component dependency.
+
+| Branch | Original version | New version |
+|--|--|--|
+| `master` | `1.3.rc1` | `1.3.0` |
+| `stable-1.3` | `1.2.1` | `1.3.0` |
+| `stable-1.2` | `1.1.3` | `1.2.2` |
+
```

Comment:
>| Branch | Original version | New version |
|--|--|--|
| `master` | `1.3.rc1` | `1.3.0` |
| `stable-1.3` | `1.2.1` | `1.3.0` |
| `stable-1.2` | `1.1.3` | `1.2.2` |

What about using this table? I took a while to figure out what was going on above.

| Branch | Original version | New version |
|--|--|--|
| `master` | `1.3.rc1` | `1.3.0` |
| `stable-1.3` | N/A| `1.3.0` |
| `stable-1.2` | `1.2.1` | `1.2.2` |
| `stable-1.1` | `1.1.3` | (unmaintained) |